mpas_analysis/default.cfg

```diff
@@ -2991,6 +2992,209 @@ compareAsContoursOnSinglePlot = True
 
 contourLevelsResult = [1027.2, 1027.4, 1027.6, 1027.7, 1027.8, 1027.85, 1027.9, 1027.95, 1028.0]
 
+[antshipTransects]
+## options related to plotting model ship CTD transects.
+
+# Times for comparison times (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
+# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
+seasons =  ['JFM']
```

Comment:
I think we probably want these to be off by default.  Maybe the easiest way to do this is with:
```suggestion
seasons =  []
```
Then, we would put the following into `polar_regions.cfg`:
```

[antshipTransects]
## options related to plotting model ship CTD transects.

# Times for comparison times (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
seasons =  ['JFM']
```
so we will get the transects in polar regions.

Reply:
I think with this one change, this PR will be good to merge.

Reply:
I have made these changes now.